### PR TITLE
alert: mention shard merging in alert's next steps

### PR DESCRIPTION
**Pull request by stefanhengl**
Shard merging reduces the number of mmapped files and is a good option to resolve this alert.

## Test plan
- just a copy change

### CODE REVIEW

**Comment by stefanhengl** [comment]
cc @daxmc99

File: doc/admin/observability/alerts.md [hunk 1]
```diff
@@ -6578,10 +6578,13 @@ Generated query for warning alert: `max((sum by (code) (increase(src_zoekt_reque
...
 - If you are running out of memory map areas, you could resolve this by:
 
+    - Enabling shard merging for Zoekt: Set SRC_ENABLE_SHARD_MERGING="1" for zoekt-indexserver. Use this option
+if your corpus of repositories has a high percentage of small, rarely udpated repositories. See
+[documentation](https://docs.sourcegraph.com/code_search/explanations/search_details#shard-merging).
     - Creating additional Zoekt replicas: This spreads all the shards out amongst more replicas, which
 means that each _individual_ replica will have fewer shards. This, in turn, decreases the
 amount of memory map areas that a _single_ replica can create (in order to load the shards into memory).
-    - Increase the virtual memory subsystem`s "max_map_count" parameter which defines the upper limit of memory areas
+    - Increasing the virtual memory subsystem`s "max_map_count" parameter which defines the upper limit of memory areas
```
Comment (daxmc99) [inline doc/admin/observability/alerts.md:6587]:
```suggestion
    - Increasing the virtual memory subsystem`s "max_map_count" parameter which defines the upper limit of memory areas, ie `sysctl sysctl -w vm.max_map_count=${NUM_REPOS}` where NUM_REPOS is a value greater than `65536` and is roughly inline with the number of repos of atleast 100MB in size
```

**Review by daxmc99 — APPROVED** [review]
LGTM

**Comment by stefanhengl** [comment]
@ggilmore Based on the comments from Dax I added recommendations for map_max_count, WDYT?

**Review by ggilmore — APPROVED** [review]

**Pull request merged**